enterprise/backend/src/metabase_enterprise/sandbox/api/table.clj

```diff
@@ -76,4 +67,19 @@
            (thunk))))
       (thunk))))
 
+(api/defendpoint GET "/:id/query_metadata"
```

Comment:
Is this wrapper still needed?

Reply:
I suspect not, but I wanted to minimized the changes. I'll follow up on this in another PR.